contrib/builder/deb/aarch64/generate.sh

```diff
@@ -97,18 +99,31 @@ for version in "${versions[@]}"; do
 			;;
 	esac
 
-	echo "# Install Go" >> "$version/Dockerfile"
-	echo "# aarch64 doesn't have official go binaries, so use the version of go installed from" >> "$version/Dockerfile"
-	echo "# the image to build go from source." >> "$version/Dockerfile"
+	case "$suite" in
+		jessie)
+			echo "# Install Go (hack: for now use pre-compiled tarball from Hypriot)" >> "$version/Dockerfile"
```

Comment:
Does debian:jessie have a go package available for arm64? Bootstrap building the GO_VERSION from source with either an older version of go, even if it had to be gccgo, would be preferable to this IMO.